### PR TITLE
Make a check for the initial value of selectionDirection less strict.

### PR DESCRIPTION
**Pull request by Ms2ger**
The standard does not define an initial value.

### CODE REVIEW

File: html/semantics/forms/textfieldselection/selection-not-application.html [hunk 1]
```diff
@@ -70,7 +75,7 @@
     }, `selectionEnd on an input[type=${type}] returns a value`);
 
     test(() => {
-      assert_equals(el.selectionDirection, "none");
+      assert_any(assert_equals, el.selectionDirection, ["forward", "backward", "none"]);
```
Comment (foolip) [inline html/semantics/forms/textfieldselection/selection-not-application.html]:
```suggestion
      assert_in_array(el.selectionDirection, ["forward", "backward", "none"]);
```

**Review by foolip — APPROVED** [review]
LGTM with nit

**Comment by tkent-google** [comment]
> The standard does not define an initial value.

I think it's defined.

https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#textFieldSelection:concept-textarea/input-selection-3
> The initial selection direction must be "none" if the platform supports that direction, or "forward" otherwise.

**Comment by foolip** [comment]
The results at https://wpt.fyi/results/html/semantics/forms/textfieldselection/selection-not-application.html might shed some light on this. @Ms2ger given @tkent-google's link, are you sure these changes are right?

**Comment by Ms2ger** [comment]
Thanks for the pointer, @tkent-google. I imagine this was tightened a bit since I made this PR. We could theoretically make the allowed values depend on the platform in the UA string or something like that, but let's not.

(If you merge this before I do, please squash.)

**Comment by foolip** [comment]
I've rebased the branch to trigger Taskcluster. It doesn't work if the branch falls too far behind.

**Comment by foolip** [comment]
Results from wpt.fyi look like an improvement across the board, merging.

**Pull request merged**